docker-app/qfieldcloud/core/serializers.py

```diff
@@ -277,9 +283,7 @@ def to_internal_value(self, data):
             if self._choices[i] == data:
                 return i
         raise serializers.ValidationError(
-            "Invalid status. Acceptable values are {}.".format(
-                list(self._choices.values())
-            )
+            "Invalid status. Acceptable values are {}.".format((self._choices.values()))
```

Comment:
Why did you change that?
```suggestion
            "Invalid status. Acceptable values are {}.".format(
            list(self._choices.values())
        )
```
Or if it is needed, can you please remove the inner braces.
```suggestion
            "Invalid status. Acceptable values are {}.".format(self._choices.values())
```

Reply:
Done